src/components/header/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import "./Header.css"
+
+function header({ image, title, description}) {
```

Comment:
React component beginnen met een hoofdletter
```suggestion
function Header({ image, title, description}) {
```